### PR TITLE
Revision for expression-bodied properties

### LINKED ISSUE #1369: C# Language keywords: get.
The read only auto implemented property feature added in C# 6 is not discussed in the migrated content.

### PR DESCRIPTION
**Pull request by rpetrusha**
# Revision for expression-bodied properties

## Summary

In C# 7, both get and set accessors can both have expression body definitions. This PR documents the change.

Fixes #1369 

## Details

The change requires changes to a number of topics, including the `get` and `set` keywords, the Properties topic, and the indexers topic.
## Suggested Reviewers

@BillWagner @mairaw

### CODE REVIEW

**Comment by rpetrusha** [comment]
This is ready for review; I think that I've revised all of the relevant topics.

**Review by BillWagner — APPROVED** [review]
This also looks great.  Let's :shipit:

File: docs/csharp/programming-guide/classes-and-structs/properties.md [hunk 1]
```diff
@@ -80,7 +94,9 @@ public string Name => First + " " + Last;
...
   
-## See Also  
+## See also
  [C# Programming Guide](../../../csharp/programming-guide/index.md)   
  [Using Properties](../../../csharp/programming-guide/classes-and-structs/using-properties.md)   
- [Indexers](../../../csharp/programming-guide/indexers/index.md)
\ No newline at end of file
+ [Indexers](../../../csharp/programming-guide/indexers/index.md)   
+ [get keyword](../../../csharp/language-reference/keywords/get.md)
+ [set keyword](../../../csharp/language-reference/keywords/set.md)
```
Comment (mairaw) [inline docs/csharp/programming-guide/classes-and-structs/properties.md]: this link is on the same line as the get keyword link in the rendered site

File: samples/snippets/csharp/programming-guide/classes-and-structs/properties-2.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   }
+
+   public string Name => $"{firstName} {lastName}";   
+}
+
+public class Example
+{
+   public static void Main()
+   {
+      var person = new Person("John", "Doe");
```
Comment (mairaw) [inline samples/snippets/csharp/programming-guide/classes-and-structs/properties-2.cs]: is this name from the CELA list?

File: samples/snippets/csharp/programming-guide/indexers/indexer-3.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+{
+   static void Main()
+   {
+      var stringCollection = new SampleCollection<string>();
+      stringCollection[0] = "Hello, World";
+      Console.WriteLine(stringCollection[0]);
+   }
+}
+// The example displays the following output:
+//       Hello, World.
```
Comment (mairaw) [inline samples/snippets/csharp/programming-guide/indexers/indexer-3.cs]: your string doesn't have a dot when you're assigning the variable

**Review by mairaw — APPROVED** [review]
Looks great! Left a few small comments.

**Pull request merged**